components/plurals/src/operands.rs

```diff
@@ -79,7 +79,8 @@ use fixed_decimal::FixedDecimal;
 ///     }),
 ///     FixedDecimal::from(12345)
 ///         .multiplied_pow10(-2)
-///         .map(|d| (&d).into())
+///         .to_string()
+///         .parse()
```

Comment:
You should be able to just `.into()` instead of `.to_string().parse()`

Reply:
when I use `.into()`, I get the following error
```
failures:

---- src/operands.rs - operands::PluralOperands (line 68) stdout ----
error[E0277]: the trait bound `Result<PluralOperands, _>: From<fixed_decimal::FixedDecimal>` is not satisfied
  --> src/operands.rs:82:10
   |
16 |         .into()
   |          ^^^^ the trait `From<fixed_decimal::FixedDecimal>` is not implemented for `Result<PluralOperands, _>`
   |
   = help: the following implementations were found:
             <Result<miniz_oxide::MZStatus, miniz_oxide::MZError> as From<&miniz_oxide::StreamResult>>
             <Result<miniz_oxide::MZStatus, miniz_oxide::MZError> as From<miniz_oxide::StreamResult>>
   = note: required because of the requirements on the impl of `Into<Result<PluralOperands, _>>` for `fixed_decimal::FixedDecimal`

error[E0283]: type annotations needed
  --> src/operands.rs:80:5
   |
14 |     FixedDecimal::from(12345)
   |     ^^^^^^^^^^^^^^^^^^ cannot infer type for type `{integer}`
   |
   = note: multiple `impl`s satisfying `fixed_decimal::FixedDecimal: From<{integer}>` found in the `fixed_decimal` crate:
           - impl From<i128> for fixed_decimal::FixedDecimal;
           - impl From<i16> for fixed_decimal::FixedDecimal;
           - impl From<i32> for fixed_decimal::FixedDecimal;
           - impl From<i64> for fixed_decimal::FixedDecimal;
           and 8 more
```

Reply:
Read the error

```
   |          ^^^^ the trait `From<fixed_decimal::FixedDecimal>` is not implemented for `Result<PluralOperands, _>`
```

Of course the trait is not implemented for `Result<PluralOperands, _>`. It is implemented for `PluralOperands`.

Try `.into().map(Ok)`

Reply:
After digging deep, I found the implementation for the reference of `FixedDecimal`, so, now it is working.